lib/screens/publish_screen.dart

```diff
@@ -384,9 +384,6 @@ class _OwnerBottomDrawerState extends State<OwnerBottomDrawer> {
                               bgColor: Colors.white.withOpacity(0.2),
                               textColor: EaselAppTheme.kWhite,
                               onPressed: () async {
-                                viewModel.videoLoadingError = '';
```

Comment:
Did you checked its purpose?

Reply:
Yes, It was for error handling in case if video player caught some error. It will reset it. But now this is getting handled in HomeScreen.

Reply:
okay.